packages/table/src/table.tsx

```diff
@@ -918,24 +919,30 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.maybeScrollTableIntoView();
     }
 
-    protected validateProps(props: ITableProps & { children: React.ReactNode }) {
-        const { children, columnWidths, numFrozenColumns, numFrozenRows, numRows, rowHeights } = props;
+    protected validateProps(prevProps: ITableProps, forceValidation: boolean = false) {
+        const { children, columnWidths, numFrozenColumns, numFrozenRows, numRows, rowHeights } = this.props;
         const numColumns = React.Children.count(children);
 
+        const shouldValidate =
```

Comment:
would be nice to leave a comment about this change for future git blames and such

Reply:
what kind of comment would you expect here?

Reply:
explain your reasoning for the behavior change, what led you to add this code?

Reply:
added a comment